src/server/engine_shard.cc

```diff
@@ -584,10 +584,10 @@ void EngineShard::PollExecution(const char* context, Transaction* trans) {
   // If any of the following flags are present, we are guaranteed to run in this function:
   // 1. AWAKED_Q -> Blocking transactions are executed immediately after waking up, they don't
   // occupy a place in txq and have highest priority
-  // 2. SUSPENDED_Q -> Suspended shards are run to clean up and finalize blocking keys
+  // 2. WAS_SUSPENDED -> Suspended transactions are run to clean up and finalize blocking keys
```

Comment:
renamed to `WAS_SUSPENDED` to denote the suspension event happened some time in the past